src/root/views/Emergency/index.js

```diff
@@ -1389,141 +1389,146 @@ class Emergency extends React.Component {
-                  >
-                    {this.renderFieldReports()}
-                  </TabContent>
-                  <TabContent
-                    isError={
-                      !get(this.props.appealDocuments, 'data.results.length')
-                    }
-                    errorMessage={strings.noDataMessage}
-                    title={strings.emergencyAppealDocumentsTitle}
-                  >
-                    {this.renderAppealDocuments()}
-                  </TabContent>
-                </TabPanel>
-                ) : null }
-
-                { this.hasRRTab() ? (
+                    <TabContent
+                      isError={!get(this.props.event, 'data.field_reports.length')}
+                      errorMessage={strings.noDataMessage}
+                      title={strings.emergencyFieldReportsTitle}
+                    >
+                      {this.renderFieldReports()}
+                    </TabContent>
+                    <TabContent
+                      isError={
+                        !get(this.props.appealDocuments, 'data.results.length')
+                      }
+                      errorMessage={strings.noDataMessage}
+                      title={strings.emergencyAppealDocumentsTitle}
+                    >
+                      {this.renderAppealDocuments()}
+                    </TabContent>
+                  </TabPanel>
+                )}
                 <TabPanel>
-                  <div>
-                    <div className='header-stats container-lg'>
-                      <div className='sumstats__wrap'>
-                        <ul className='sumstats'>
-                          <li className='sumstats__item__wrap'>
-                            <div className='sumstats__item'>
-                              <img className='sumstats__icon_2020' src='/assets/graphics/layout/heops-brand.svg' />
-                              <span className='sumstats__value'>
-                                {n(this.props.aggregated.data.active_deployments)}
-                              </span>
-                              <Translate className='sumstats__key' stringId='deploymentsOngoingRR'/>
-                            </div>
-                          </li>
-                          <li className='sumstats__item__wrap'>
-                            <div className='sumstats__item'>
-                              <img className='sumstats__icon_2020' src='/assets/graphics/layout/eru-brand.svg'/>
-                              <span className='sumstats__value'>
-                                {n(this.props.aggregated.data.active_erus)}
-                              </span>
-                              <Translate className='sumstats__key' stringId='deploymentsDeployedERU'/> &nbsp;
-                            </div>
-                          </li>
-                        </ul>
+                  <Activities
+                    emergencyId={this.props.match.params.id}
```

Comment:
don't we need to check match or params for undefined?

Reply:
Nope, the page will break long before it gets here